db/migrate/20171210195325_create_tenants.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTenants < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tenants do |t|
+      t.string :name
+      t.string :phone
+      t.string :website
+      t.string :logo
```

Comment:
tenant should have owner_id or user_id (of owner of this tenant)

Reply:
What do you exactly mean? Should I add foreign_key for tenants table?